### DISCUSSION TITLE
How to get rid of the Short or Download button?

### DISCUSSION THREAD

**Discussion by Aholicknight** [PoomSmart/YTABConfig#26] [Q&A]
![image](https://user-images.githubusercontent.com/7843719/199099804-8b1be4f2-a65a-4aba-8cd7-4c3515c60090.jpeg)

I am trying to get rid of the download button and sometimes the clip/short button shows up as well. I have "hide download, shorts, and clip" already disabled in uYou but they do not go away. Thank you!

**Comment by CySxL** [comment]
> ![image](https://user-images.githubusercontent.com/7843719/199099804-8b1be4f2-a65a-4aba-8cd7-4c3515c60090.jpeg)
> 
> I am trying to get rid of the download button and sometimes the clip/short button shows up as well. I have "hide download, shorts, and clip" already disabled in uYou but they do not go away. Thank you!

Those features of hiding buttons under video player only works with Legacy features on.

But yes I would want them being removed as well. So annoying and useless.

**Comment by arichornlover** [comment]


**Reply by floyega** [reply]
Speak of the devil… 👹

**Reply by arichornlover** [reply]
Well I’m not always right ☹️
